lib/internal/util/inspect.js

```diff
@@ -2000,6 +2011,85 @@ function stripVTControlCharacters(str) {
   return str.replace(ansi, '');
 }
 
+function getPrivatePropertyPreviews(ctx, target) {
+  const preview = previewValue(target);
+  if (preview == null) {
+    return undefined;
+  }
+  const { privateProperties } = preview;
+  if (privateProperties == null) {
+    return undefined;
+  }
+  return privateProperties.map((it) => formatPrivatePropertyPreview(ctx, it));
+}
+
+function previewPrelude() {
+  if (privateInspectionSession == null) {
+    inspector = require('inspector');
+    privateInspectionSession = new inspector.SyncSession();
+    privateInspectionSession.connect();
+  }
+}
+
+function previewValue(/** used */target) {
+  previewPrelude();
+  let previews;
+  privateInspectionSession.post('Runtime.enable');
+  privateInspectionSession.post('Debugger.enable');
+  privateInspectionSession.once('Debugger.paused', (event) => {
```

Comment:
Could we just have like a `globalThis.__inspect_target` that we create and use and then delete, so we don't have to deal with the huge perf hit from forcing evaluation through the debug interpreter?